src/docs/integrations/jira/index.mdx

```diff
@@ -1,18 +1,27 @@
 ---
-title: "Jira Integration"
+title: 'Jira Integration'
 ---
 
-Create an [Atlassian Cloud Instance](http://go.atlassian.com/cloud-dev)
+Create an [Atlassian Cloud Developer Instance](http://go.atlassian.com/cloud-dev)
 
-Configure Jira settings.
-  1. Navigate to Jira settings (cog icon) > Apps > Manage apps.
-  2. Scroll to the bottom of the Manage apps page, and click Settings (1).
-  3. Select Enable development mode and Enable private listings (2), and click Apply.
+Atlassian offers a development-specific org option, different from simply having a personal Jira organization. Even if you already have a personal Jira org, it's worth creating a development org, because it will correctly set a number of necessary options for you automatically.
+
+Set Up ngrok
+
+(This is only necessary if you're developing locally.) [Download](https://ngrok.com/download) and [set up](https://ngrok.com/docs/getting-started/) ngrok, then start an ngrok server.
+
+Configure Jira Settings
+
+1. Create a new project in Jira.
+2. In the navigation bar at the top of the screen, go to Apps > Manage Your Apps.
+3. At the bottom of the Manage Apps page, click Settings (1).
+4. Select Enable Development Mode and Enable Private Listings (2), and click Apply.
 
 ![Jira Settings](./configure-jira.png)
 
-Upload the Atlassian Connect descriptor to Jira. It can be found at `{YOUR_DOMAIN}/extensions/jira/descriptor/`
+5. Reload the page, and Upload App should appear next to Build a New App.
+6. Click Upload App and upload the Atlassian Connect descriptor by using the URL  `https://{YOUR_SENTRY_DOMAIN}/extensions/jira/descriptor/`. Note that if you are running a local devserver, `YOUR_SENTRY_DOMAIN` will be your ngrok domain.
```

Comment:
It's possible to use other services / methods besides ngrok here which is why we've kept the language agnostic

Reply:
I can add "(or equivalent)" if that helps. More or less I'm trying to write down all of the things I had to figure out/realize so that they're just obvious for the next person.

Reply:
Okay, I updated it to, "Note that if you are running a local devserver, `YOUR_SENTRY_DOMAIN` will be your ngrok (or other tunneling service) domain."